src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1386,6 +1395,8 @@ public void run() {
         });
         logCopyErrorThread.start();
 
+        containerStartTimestamp = Timestamp.from(Instant.now());
+        writeDevcMetadata();
```

Comment:
Would you ever call `writeDevcMetadata` without first setting a start or stop time immediately before? It seems like they always go together. So maybe combine them in one method? You could have a `createDevcMetadata(boolean start)` and if the `start` flag is true create it with a start timestamp, otherwise create it with a stop timestamp.  Actually, the timestamp seems irrelevant. We never actually look at the value to determine something. It's just whether it is there or not.